app/helpers/policy_disregards_helper.rb

```diff
@@ -0,0 +1,78 @@
+module PolicyDisregardsHelper
+  include CheckAnswersHelper
+
+  ATTRIBUTES = %i[
+    england_infected_blood_support
+    vaccine_damage_payments
+    variant_creutzfeldt_jakob_disease
+    criminal_injuries_compensation_scheme
+    national_emergencies_trust
+    we_love_manchester_emergency_fund
+    london_emergencies_trust
+  ].freeze
+
+  def policy_disregards_hash(policy_disregards)
+    items = policy_disregards_as_array(policy_disregards)
+    items&.compact!
+    return nil unless items.present?
+
+    {
+      items: items
+    }
+  end
+
+  def policy_disregards_as_array(policy_disregards)
+    ATTRIBUTES&.map do |attribute|
```

Comment:
why is the `&` needed here?

Reply:
It's syntax to make sure there are values there before mapping over (Colin says it's called the safe navigation operator)

Reply:
Ah cool, I know it's used for digging, but good to know 👍

Reply:
But if it's a constant, I think you can be pretty sure it's there, no?